ml-agents/mlagents/trainers/tests/test_environments/test_simple.py

```diff
@@ -94,7 +103,7 @@ def reset(
         self._reset_agent()
 
         agent_info = AgentInfoProto(
-            stacked_vector_observation=[self.position] * OBS_SIZE,
+            stacked_vector_observation=[self.goal] * OBS_SIZE,
```

Comment:
Shouldn't the vector observation also include the position in some way (maybe concat the position and goal)? I suppose it will still train if just the goal is given by taking the same action always.

Reply:
The way this is set up, the agent always starts at 0 and gets a reward at +/- 1, so the "best" policy is always action=observation[0] (for continuous) or (0 if observation[0] < 0 else 1) for discrete.

Reply:
Got it. So it should train either way.